api/capi/include/nnstreamer.h

```diff
@@ -238,6 +244,19 @@ typedef void (*ml_pipeline_sink_cb) (const ml_tensors_data_h data, const ml_tens
  */
 typedef void (*ml_pipeline_state_cb) (ml_pipeline_state_e state, void *user_data);
 
+/**
+ * @brief Callback for custom condition of tensor_if
+ * @since_tizen 6.5
+ * @param[in] info The handle of tensors information (cardinality, dimension, and type of given tensor/tensors).
+ * @param[in] data The handle of the tensor output of the pipeline (a single frame. tensor/tensors). Number of tensors is determined by ml_tensors_info_get_count() with the handle 'info'. Note that the maximum number of tensors is 16 (#ML_TENSOR_SIZE_LIMIT).
+ * @param[in/out] user_data User application's private data.
+ * @param[out] Result of the user-defined condition. 0 refers to FALSE and a non-zero value refers to TRUE.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
```

Comment:
IMO, this is user callback, so ml error code is useless.

Reply:
Right. ml error code is useless. updated.